pycbc/events/events.py

```diff
@@ -315,12 +317,9 @@ def keep_loudest_in_interval(self, window, num_keep):
     def add_template_events(self, columns, vectors):
         """ Add a vector indexed """
         # initialize with zeros - since vectors can be None, look for the
-        # first one that isn't
+        # longest one that isn't
         new_events = None
-        for v in vectors:
-            if v is not None:
-                new_events = numpy.zeros(len(v), dtype=self.event_dtype)
-                break
+        new_events = numpy.zeros(max([len(v) for v in vectors if v is not None]), dtype=self.event_dtype)
```

Comment:
Why was this change necessary?

Reply:
I can't remember. I've reverted it for now, and hopefully it won't break anything later on.

Reply:
I remembered why this is necessary. Some of the pycbc multi inspiral outputs are 'None' or only have one value per template/segment. Others have one value per trigger. When these get written to hdf, if there is only one value, then it gets written out repeatedly (once for each trigger). 

I can probably fix this in the executable if you prefer (all the outputs have the same length when it reaches this point). Or we can make the change posted above.